scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1274,15 +1282,18 @@ private void build() {
         setEnabledStartAtPart(false);
 
         this.window = new Window();
-        this.window.setTitle("Submit a new job");
+        this.window.setTitle(isResubmit ? "Re-submit job " + jobId : "Submit a new job");
```

Comment:
Why do we need to set the title in case of re-submit? I thought that the window should not show up for a resubmission.

Reply:
@jrochas In fact I was using a forked (dedicated) window only for dev. Since the code would have been 80% identical I only updated the Submit Window for resubmissions. Otherwise there would have been an almost duplicate class.